public/js/admin.js

```diff
@@ -436,7 +436,10 @@ var updateVisibility = function(qid) {
         },
         success: function(data) {
             displayQuestionTable();
-            dropSnack(colours.SUCCESS_GREEN, 'Question ' + qid + ' is now visible to the students');
+
```

Comment:
No need for new lines here. Also both options say visible to the student. Am I missing something or shouldn't the second option say not visible?

Reply:
I see the invisible. It would be better to say not visible, I barely noticed the (in)visible.

Reply:
@ahmedqarmout2 Yessir, sounds good. What if i wrote "visible" and "not visible" and i made the former a blue snack/toast and the latter a yellow snack/toast? Ill try to add an icon but I'm not sure if it would mess with the snack/toast's animations in any way.

What does everyone think?

Reply:
Sounds great, except the blue color XD why blue? I mean if there is a meaning for blue then sure yeah, blue seems odd XD

Reply:
@ahmedqarmout2 LOL cause I couldn't think of any other colour. I wanna have different colours for "visible" and "not visible" so its more apparent. Suggestions?

Reply:
Go with green instead of blue.

Reply:
Same green as the success green or a different one?

Reply:
Same

Reply:
Green for showing a question cuz that's a success sign, and there is nothing wrong with showing a question. 
Yellow for hiding the questions as this is a warning for the prof saying hey watch out students won't see this question 
Makes sense?

Reply:
:+1: Si senor. Arigato :)